admin/sql/create_indexes.sql

```diff
@@ -7,4 +7,6 @@ CREATE UNIQUE INDEX data_sha256_ndx_lowlevel ON lowlevel (data_sha256);
 CREATE INDEX mbid_ndx_highlevel ON highlevel (mbid);
 CREATE INDEX build_sha1_ndx_highlevel ON lowlevel (build_sha1);
 
+CREATE INDEX lower_musicbrainz_id_ndx_user ON "user" (lower(musicbrainz_id));
```

Comment:
That's actually one of the things I was worried about. Right now we are inserting from only one place, but it seems like a good idea to take care of this.